### DISCUSSION TITLE
Is there any easy way to customize styles?

### DISCUSSION THREAD

**Discussion by Get-108** [umputun/remark42#1634] [Q&A]
Thank you for great system.
Everything is working fine but i couldn't find any workaround to change frontend styles.
Actually i need to change green link/button style to another color.
Is it feasible with reasonable efforts?)
Thank you!

**Comment by DejavuMoe** [comment]
I have a similar idea recently, I noticed that someone has already made a PR #1633  for custom colors or themes. I see that the PR has not been merged yet, the repository maintainers may spend more time to merge this "custom" features.

An easy way to implement this is to modify the Remark42 [front-end style](https://github.com/umputun/remark42/tree/master/frontend/apps/remark42/app/styles)，and then just build the docker image.

**Comment by mprokopov** [comment]
Here is one of ideas for simplified theme customisation, though, not always possible.

In case you do self-hosted deployment, Remark42 could be run on the same subdomain as your site. This would allow remark widget customisation using your own styles right from your main site due to SAME ORIGIN policy. 

Unfortunately, it doesn't work if remark itself runs on different domain.

**Comment by billy-le** [comment]
I recently went into customizing Remark42 and it seems you can pass a `__colors__` object into the `remark_config` variable.

The issue with this approach is that it doesn't work for dark themes. I'm figuring out how to do that with the remark base image from DockerHub but have no luck.
